oss-cloud-backend/gitHubApiService.js

```diff
@@ -30,7 +27,7 @@ module.exports.getForkedRepos = async(username) => {
 
 // makes a call to github api for each repo in array, returning a more detailed representation of the repo
 module.exports.getRepoDetails = async (repos) => {

```

Comment:
There is no need for `return await Promise...`.
`await` keyword can be omitted in return statement of `async` function. [Link to StackOverflow](https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise)